src/modules/createtr.js

```diff
@@ -0,0 +1,39 @@
+const renderContainer = async (gameLInk) => {
+  const res = await fetch(gameLInk);
+  await res.json()
+    .then((resData) => {
+      const container = document.querySelector('.table');
+      container.replaceChildren();
+      let counts = 1;
+      resData.result.forEach((scores, index) => {
+        container.innerHTML += `
+        <li>
+          <p>${scores.user}:</p>
+          <p>${scores.score}</p>
+        </li> `;
+        container.classList.add('x');
+        if (counts % 2 === 0) {
+          container.children[index].classList.add('color');
+        } else {
+          container.children[index].classList.add('no-color');
+        }
+
+        counts += 1;
+      });
+    });
```

Comment:
The point of using `async/await` is to avoid `Promises` which involves using the `.then` keyword to access data from an API.

- On line 3, you can do something like `const resData = await res.json();`. The `resData` variable should have all the data you need that you can use on the next block without using `then`.

Reply:
thank you it has been fixed